recipes/libspng/all/conandata.yml

```diff
@@ -1,11 +1,18 @@
 sources:
+  "0.7.4":
+    url: "https://github.com/randy408/libspng/archive/refs/tags/v0.7.4.tar.gz"
+    sha256: "47ec02be6c0a6323044600a9221b049f63e1953faf816903e7383d4dc4234487"
   "0.7.3":
     url: "https://github.com/randy408/libspng/archive/refs/tags/v0.7.3.tar.gz"
     sha256: "a50cadbe808ffda1a7fab17d145f52a23b163f34b3eb3696c7ecb5a52340fc1d"
   "0.7.2":
     url: "https://github.com/randy408/libspng/archive/refs/tags/v0.7.2.tar.gz"
     sha256: "4acf25571d31f540d0b7ee004f5461d68158e0a13182505376805da99f4ccc4e"
 patches:
+  "0.7.4":
+    - patch_file: "patches/0.7.4-0001-allow-miniz.patch"
+      patch_description: "add miniz option which is written in docs/BUILD.md"
```

Comment:
```suggestion
      # Miniz is supported by the project, but does not currently expose a CMake
      # option to toggle it. This can be removed once the project supports it
      # or when the recipe moves to use Meson instead
      patch_description: "add miniz option which is written in docs/BUILD.md"
```